app/controllers/miq_ae_class_controller.rb

```diff
@@ -1749,7 +1753,7 @@ def domains_priority_edit
       domains = @edit[:new][:domain_order].reverse!.collect do |domain|
         MiqAeDomain.find_by_name(domain.split(' (Locked)').first).id
       end
-      MiqAeDomain.reset_priority_by_ordered_ids(domains)
+      MiqAeDomain.reset_priority_by_ordered_ids(domains, current_tenant)
```

Comment:
@dclarizio You mentioned in a previous conversation that you had a concern about explicitly passing current_tenant here and maybe the MiqAeDomain model should use `User.current_tenant`.  I tend to like this method better than relying on the User model.  Thoughts?